### ISSUE TITLE
Check filters updates does not work

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdguardDNS#317]
It always returns "OK 0 filters updated" immediately even though I have a filter added.

**Comment by hmage** [comment]
This is because `filtering/add_url` returns only after downloading the filter. So immediate request to `filtering/refresh` will return 0 because `filtering/add_url` just downloaded it and refresh will not redownload just-downloaded url.

@maximtop @IldarKamalov -- could you guys please add confirmation dialog after `filtering/add_url` returns successfully with text 'filter downloaded successfully`?

**Comment by maximtop** [comment]
```
This is because filtering/add_url returns only after downloading the filter. So immediate request to filtering/refresh will return 0 because filtering/add_url just downloaded it and refresh will not redownload just-downloaded url.
```
@hmage seems that @ameshkov reported about check updates button
https://uploads.adguard.com/mtopciu_uhhod.png

as you can see filter is already added, but when I press button check updates, i get only OK 0 filters updated, http://localhost:3000/control/filtering/refresh

I have checked with the copy of filter posted on github pages https://maximtop.github.io/test-filters-update/15.txt. I've posted old old version of filter, added it to the dns-adguard, in a while i've updated this filter, and pressed button to update filters but I've received that '0 filters were updated' 
Also I've noticed that there were no log messages in the terminal

**Comment by hmage** [comment]
Fixed in 81c7dbbc1684622e8b2e05d39dcd4605339ff24f
